pyramid_mailer/tests/test_message.py

```diff
@@ -24,7 +20,7 @@ def test_data_from_string(self):
         self.assertEqual(a.data, "foo")
 
     def test_data_from_file_obj(self):
-        a = self._makeOne(data=StringIO(text_type("foo")))
+        a = self._makeOne(data=StringIO(str("foo")))
```

Comment:
Seems redundant to run str() on a string...

Reply:
Yep it does. Some automated assistance was used in fixing this stuff. For small changes like this github suggestions can be a good way of providing feedback that can be merged from the interface. e.g. 

```suggestion
        a = self._makeOne(data=StringIO("foo"))
```

Reply:
Merging from the PR means lots of tiny commits with what amounts to bad commit messages that never seem to get rebased back in, which is why I tend to not use that, so that we can keep clean git commit history since we don't squash on merge.